src/components/dialogs/ConfirmDissolution.vue

```diff
@@ -100,6 +100,11 @@ export default class ConfirmDissolution extends Vue {
     return this.getDissolutionConfirmationResource?.dissolutionType
   }
 
+  /** The incorporation text to display. */
+  private get incorporationText (): string {
```

Comment:
Small cleanup: please change the local getters in this file to public (just remove private).

Also please change the prop to readonly.

Reply:
done